src/cryptoadvance/specter/server_endpoints/wallets.py

```diff
@@ -1067,6 +1067,15 @@ def decoderawtx(wallet_alias):
 
             if tx["confirmations"] == 0:
                 tx["is_purged"] = wallet.is_tx_purged(txid)
+                try:
+                    if wallet._transactions[txid].get("category", "") == "receive":
+                        tx["fee"] = (
+                            wallet.rpc.getmempoolentry(txid)["fees"]["modified"] * -1
+                        )
+                except Exception as e:
+                    app.logger.warning(
```

Comment:
can you either do one of:
* Catch a more specific Exception (`RpcError`)
* `handle_exception(e)` 
* `logger.exception(e)`

This combination is imho suboptimal.

Reply:
Added handle_exception(e)